ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_from_shape_or_value.py

```diff
@@ -234,6 +234,33 @@ def test_numpy_fromfunction(
     )
 
 
+@handle_frontend_test(
+    fn_tree="numpy.fromiter",
+    iterable=st.lists(st.floats(), min_size=1, max_size=10),
+    dtype=helpers.get_dtypes("numeric"),
+)
+def test_numpy_fromiter(
+    iterable,
+    dtype,
+    frontend,
+    backend_fw,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    helpers.test_frontend_function(
+        iterable=iterable,
```

Comment:
the "iterable" here should be the same as your function's first arguement "iter"

Reply:
Sorry about that, fixed.